tests/suites/test_suite_ssl.function

```diff
@@ -1325,7 +1479,7 @@ void ssl_message_queue_interleaved( )
 
     mbedtls_test_message_queue_setup( &queue, 3 );
 
-    /* Interleaved test - 2 pushes, 1 or 2 pops a couple of times
+    /* Interleaved test - [2 pushes, 1 pop] twice, and then two pops
```

Comment:
This should be added in "Add a message metadata queue in ssl tests", not "Add a message-based socket mock connection to the ssl tests"